index.html

```diff
@@ -217,7 +220,11 @@ <h2>Rooms</h2>
       </header>
       <div class="room">
          <div>
+            <a href="#">

```

Comment:
Just wrapping the images in an `img` tag will do the trick. No need for the paragraph with the class of pointer.
### Todo

- [ ] Wrap the images with the anchor tag
- [ ] Remove the paragraph tag
- [ ] Remove the pointer class styling because links have pointers by default